pkg/model/config/config.go

```diff
@@ -87,10 +70,10 @@ func (config Config) HasResource(target GVK) bool {
 
 // AddResource appends the provided resource to the tracked ones
 // It returns if the configuration was modified
-// NOTE: this works only for v2, since in v1 resources are not tracked
+// NOTE: in v1 resources are not tracked, so we return false
```

Comment:
```suggestion
```

Reply:
As I said above, this check is needed so the note has to stay.